.github/workflows/release_manager.yml

```diff
@@ -0,0 +1,59 @@
+name: Release Manager
+
+on: 
+  push:
+    tags:
+      - "v[0-9]"
+
+jobs:
+  release_manager:
+    name: Release Manager
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-20.04, macos-latest]
```

Comment:
What about windows?

Reply:
We don't release Windows on CI, we have to do it manually because we use the hardware signing token.

Reply:
We should still produce the binary, I think?

Reply:
We sign and build the binary at the same time, so we can't produce the binary on CI.